libraries.json

```diff
@@ -572,22 +569,22 @@
   {
     "github": "chRyNaN/chords",
     "category": "UI",
-    "maven": "https://dl.bintray.com/chrynan/chrynan/com/chrynan/chords/chords-core/"
+    "maven": "https://repo.repsy.io/mvn/chrynan/public/com/chrynan/chords/"
```

Comment:
here no library at all.
should be https://repo.repsy.io/mvn/chrynan/public/com/chrynan/chords/chords-core/